macOS/LocalPackages/DataBrokerProtection/Tests/DataBrokerProtectionTests/CaptchaServiceTests.swift

```diff
@@ -45,7 +45,7 @@ final class CaptchaServiceTests: XCTestCase {
         MockURLProtocol.requestHandlerQueue.append({ _ in throw MockError.someError })
         let sut = CaptchaService(urlSession: mockURLSession,
                                  authenticationManager: mockAuthenticationManager,
-                                 settings: DataBrokerProtectionSettings(defaults: .standard),
+                                 settings: DataBrokerProtectionSettings(defaults: .standard, proxySettings: .init(defaults: .standard)),
```

Comment:
Is `UserDefaults.standard` shared across tests?

If so, it might be worth considering to move over `UserDefaults(suiteName: UUID().stringValue)` to avoid cross-test contamination.